kani-driver/src/args/mod.rs

```diff
@@ -886,4 +962,12 @@ mod tests {
         assert_eq!(parse(&["kani", "--features", "a", "--features", "b,c"]), ["a", "b", "c"]);
         assert_eq!(parse(&["kani", "--features", "a b", "-Fc"]), ["a", "b", "c"]);
     }
+
+    #[test]
+    fn check_kani_playback() {
+        let input = "kani playback --test dummy file.rs".split_whitespace();
+        let args = StandaloneArgs::try_parse_from(input).unwrap();
+        assert_eq!(args.input, None);
```

Comment:
Why is `args.input` `None`? Shouldn't it be `Some("file.rs")`?

Reply:
Tbh, this parsing code is not ideal. It is a bit painful to handle positional arguments and subcommands reliably, especially in our case where the subcommand is optional.

The way I implemented, I tried to isolate the subcommand parser from the verification parser. So internally there are two `input` fields. The first one is inside StandaloneArgs and it's only set when no subcommand is provided. The second is inside `KaniPlaybackArgs` which is only set when user selects the playback subcommand.

It's possible that using global option would work, but the post parsing validation and the entire playback would need knowledge of the global options.

Reply:
Oh, and I did ignore the toml arguments. o.O